new_course/1_programming_fundementals/debugging.md

```diff
@@ -125,9 +125,31 @@ end
 double_words_in_phrase("This is a test")
 ~~~
 
-When your code executes and gets to `binding.pry`, it will open an IRB-like session in your terminal. You can then use that session to check the values of anything within the scope of where you included `binding.pry`.
+When your code executes and gets to `binding.pry`, it will open an IRB-like session in your terminal. You can then use that session to check the values of anything within the scope of where you included `binding.pry`. However, keep in mind that any code written *after* the `binding.pry` statement will not have been evaluated during the Pry session. In other words, when we add a `binding.pry` statement, we are effectively creating a breakpoint or a pause in our code.
```

Comment:
I'm not sure that the second sentence is that helpful in putting this another way. Perhaps 
"Thus, adding a `binding.pry` line is similar to adding a breakpoint in JavaScript." ? If someone isn't familiar with breakpoints in JS, this might go down smoother.